src/object/srv_obj_migrate.c

```diff
@@ -1065,18 +1066,31 @@ migrate_fetch_update_single(struct migrate_one *mrone, daos_handle_t oh,
 				       mrone->mo_epoch, ORF_EC,
 				       start_shard, 1,
 				       true, false, NULL);
+		los[i].cs_even_dist = 1;
+		los[i].cs_bytes = obj_ec_singv_cell_bytes(
+					mrone->mo_iods[i].iod_size, oca);
+		los[i].cs_nr = obj_ec_tgt_nr(oca);
```

Comment:
for small singv it is stored on single target, should it be handled as well?

Reply:
ah, yes, small singv should use fetched csum directly.